.github/workflows/ci.yaml

```diff
@@ -1,5 +1,9 @@
 name: CI
-on: pull_request
+on:
+  pull_request:
+    paths:
+    - "site/**"
+    - "!site/en-snapshot/**"
```

Comment:
Is the `!` right?

Reply:
Yeah. Looks a little strange but seems prefered in their docs: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-positive-and-negative-patterns-1